### DISCUSSION TITLE
Stuck with the advanced config for power back to grid and solar

### DISCUSSION THREAD

**Discussion by mmmgggttt** [emporia-vue-local/esphome#253] [Q&A]
Hello community, i've successfully compiled the config of the standard version (including the 3rd phase as i'm 380v 3 phases in italy)
but when i add the "optional" config i got several compile error, such as duplicate sensor and when i remove sensor, many other in the indentation. I'm sure this is a very noob issues (As i'm new to such coding activities) but i can't get round that.
Any help ? than kyou !

`sensor:
  - platform: emporia_vue
    ct_clamps:
      - phase_id: phase_a
        input: "A"  # Verify the CT going to this device input also matches the phase/leg
        power:
          name: "Phase A Power Return"
          id: phase_a_power_return
          filters: [*throttle_avg, *invert]  # This measures energy uploaded to grid on phase A
      - phase_id: phase_b
        input: "B"  # Verify the CT going to this device input also matches the phase/leg
        power:
          name: "Phase B Power Return"
          id: phase_b_power_return
          filters: [*throttle_avg, *invert]  # This measures energy uploaded to grid on phase B
  - platform: template
    name: "Total Power Return"
    lambda: return id(phase_a_power_return).state + id(phase_b_power_return).state;
    update_interval: 1s
    id: total_power_return
    device_class: power
    state_class: measurement
    unit_of_measurement: "W"
  - platform: total_daily_energy
    name: "Total Daily Energy Return"
    power_id: total_power_return
    accuracy_decimals: 0`

and 

`sensor:
  - platform: template
    name: "Solar Power"
    lambda: return id(cir15).state + id(cir16).state;
    id: solar_power
    device_class: power
    state_class: measurement
    unit_of_measurement: "W"
  - platform: total_daily_energy
    name: "Solar Daily Energy"
    power_id: solar_power
    accuracy_decimals: 0`

**Comment by jasonogren** [comment]
You don't need 'sensor:' twice. Remove the 'sensor:' line before the "solar power" sensors and just make sure the indentations for the 'platforrm: template' lines follow the sensor entries from above.
